src/AcceleratedDubins.jl

```diff
@@ -500,63 +449,43 @@ Sample single path for plotting
 function sample_path(path::DubinsPathR2, resolution::Float64 = 0.01)
     if path == nothing
         return [], []
```

Comment:
This returns type `Any[]`, thus the return type of the `sample_path` may contain type instability.

Reply:
Remained in the code from original implementation. I'll fix in the code.
```suggestion
        return Vector{Float64}(), Vector{Float64}()
```